concent_api/core/queue_operations.py

```diff
@@ -0,0 +1,2 @@
+def send_verification_request():  # TODO implement sending real verivication_request to Work Queue
```

Comment:
Is there going to be something more or this is the part when Conductor starts ?

Reply:
this is the part where Conductor starts, so when your part is in `master`, I will create and send a real `verification_request` here ;)